src/modules/Search/Search.js

```diff
@@ -166,6 +166,11 @@ export default class Search extends Component {
     /** A search can have its results take up the width of its container. */
     fluid: PropTypes.bool,
 
+    /** A search input can take up the width of its container . */
+    input: PropTypes.shape({
```

Comment:
I have changed this to customPropTypes.itemShorthand but I don't follow you about Input.create method. I am getting this error when using create()
![screenshot from 2017-01-06 16-39-37](https://cloud.githubusercontent.com/assets/9512969/21716164/c305f920-d42e-11e6-8bf6-7ffb46293030.png)